oio/api/object_storage.py

```diff
@@ -705,6 +706,7 @@ def object_create_ext(self, account, container, file_or_path=None,
         """
         if (data, file_or_path) == (None, None):
             raise exc.MissingData()
+        kwargs.setdefault('reqid', request_id('create-'))
```

Comment:
At this point, there is already a request ID, set by the `ensure_request_id`. If you want the generated request ID to be prefixed by something, replace the decorator by `ensure_request_id2` and pass the prefix as an argument.